src/chains/algorand_1/examples/transfers.ts

```diff
@@ -38,12 +38,23 @@ const composeTokenTransferParams: Partial<AlgorandActionAssetTransferParams> = {
 }
 
 const composeAlgoPaymentParams: Partial<AlgorandActionPaymentParams> = {
-  from: 'VBS2IRDUN2E7FJGYEKQXUAQX3XWL6UNBJZZJHB7CJDMWHUKXAGSHU5NXNQ',
+  from: 'WCZV75K44NWDFWTYJCCOJ6NZC2TYAODLIC7GFMMXRKY2JNPW3LTB4IUL3U',
   to: 'TF6PJW7VSEKD5AXYMUXF5YGMPDUWBJQRHH4PYJISFPXAMI27PGYHKLALDY',
   note: 'transfer memo',
   amount: 1,
 }
 
+// requires transaction setup with betanet endpoints and accounts.
```

Comment:
This should have a date and a reason. e.g. As of Aug 2020 - Rekey feature only available on beta test net - use beta endpoints to run